patch-package
--- a/node_modules/@uirouter/core/lib/vanilla/baseLocationService.d.ts
+++ b/node_modules/@uirouter/core/lib/vanilla/baseLocationService.d.ts
@@ -40,6 +40,6 @@ export declare abstract class BaseLocationServices implements LocationServices,
     path: () => any;
     search: () => any;
     url(url?: string, replace?: boolean): string;
-    onChange(cb: EventListener): () => Function[];
+    onChange(cb: Function): Function;
     dispose(router: UIRouter): void;
 }
